src/main/java/org/mskcc/limsrest/controller/GetSequencingRequests.java

```diff
@@ -17,7 +17,7 @@
 import java.util.List;
 import java.util.Map;
 
-import static com.velox.api.datarecord.DataFieldHelper.parseInteger;
+import static com.velox.api.datarecord.DataFieldHelper.parseLong;
```

Comment:
Could just use Long.parseLong() instead of import?

Reply:
will do